src/main/resources/hudson/plugins/jira/Messages.properties

```diff
@@ -26,3 +26,6 @@ VersionReleaser.AlreadyReleased=[JIRA] The version {0} is already released in pr
 VersionReleaser.MarkingReleased=[JIRA] Marking version {0} in project {1} as released.
 JiraVersionCreator.VersionExists=[JIRA] A version with name {0} already exists in project {1}, so nothing to do.
 JiraVersionCreator.CreatingVersion=[JIRA] Creating version {0} in project {1}.
+JiraEnvironmentVariableBuilder.DisplayName=Add JIRA related environment variables to build
```

Comment:
Change to below, to fix dropdown ordering:
"JIRA: Add related environment variables to build"